### ISSUE TITLE
afterFind 2if ($lang == $this->defaultLanguage)

### ISSUE THREAD

**Issue by Vashkatsi** [OmgDef/yii2-multilingual-behavior#82]
https://github.com/OmgDef/yii2-multilingual-behavior/blob/9589c9dfdf5f082b29a0b24f48bdc68c3c23c27f/src/MultilingualBehavior.php#L291

I think this line is a mistake, you should change $this->defaultLanguage to $this->getCurrentLanguage()

**Comment by Webkadabra** [comment]
I don't think it should. This line makes sure to set the base attribute when it should
